README.rst

```diff
@@ -24,6 +24,7 @@ Design Principles
 *  Allow module development in any dynamic language, not just Python.
 *  Be usable as non-root.
 *  Be the easiest IT automation system to use, ever.
+* It is a agent-less tool so easy to use and configure
```

Comment:
On line 17 we already have
> Avoid custom-agents and additional open ports, be agentless by
   leveraging the existing SSH daemon.	   leveraging the existing SSH daemon.